cmd/amplifier/server/configuration.go

```diff
@@ -10,17 +10,17 @@ import (
 )
 
 const (
-	DefaultPort          = ":50101"
-	DefaultPublicAddress = "local.appcelerator.io"
-	DefaultTimeout       = time.Minute
+	DefaultPort       = ":50101"
+	DefaultPublicHost = "127.0.0.1"
```

Comment:
the way it's done most of the time is a URL, when the port is not explicitly set the CLI knows which port to try. We can image that the CLI first try TLS/443, and falls back to non TLS/50101. Unless the scheme and port are explicitly set.